### DISCUSSION TITLE
add command for  pm2 cluster npx nuxi start

### DISCUSSION THREAD

**Discussion by andysay** [nuxt/nuxt#16034] [Ideas]
hello, i want use pm2 cluster mode for nuxt3 , and command start not support now?

**Comment by danielroe** [comment]
Nuxt 3 uses Nitro to produce a standalone server, which is executed directly `node .output/server/index.mjs`.

[More info](https://v3.nuxtjs.org/concepts/server-engine#standalone-server).

**Reply by leonyangdev** [reply]
but，how to use pm2 manage nuxt3 project in prod？

**Comment by andysay** [comment]
```
module.exports = {
  apps: [
    {
      name: 'NuxtAppName',
      exec_mode: 'cluster',
      instances: 'max', // Or a number of instances
      script: './.output/server/index.mjs',
      args: 'start'
    }
  ]
}
```

working solution
